src/foam/nanos/auth/ResendVerificationEmail.js

```diff
@@ -104,11 +105,7 @@ foam.CLASS({
       name: 'goBack',
       label: 'Go to sign in page.',
       code: function(X) {
-        this.auth.logout().then(function() {
-          this.window.location.hash = '';
-          this.window.location.reload();
-          localStorage.removeItem('defaultSession');
-        });
+        this.pushMenu('sme.accountProfile.signout');
```

Comment:
sme and accountProfile do not exist in foam.
Lets move this signout menu to foam or create the minimal equivalent on the foam side. 
Thanks

Reply:
Replaced nanopay logout menu with foam one